docs/mkdocs.yml

```diff
@@ -66,3 +66,7 @@ nav:
     - Meta Functions: result/metafunctions.md
     - Comparisons: result/comparisons.md
     - CHANGELOG: result/CHANGELOG.md
+  - Anyhow Module:
+    - anyhow 101: anyhow/anyhow.md
+  - Thiserror Module:
+    - anyhow 101: thiserror/thiserror.md
```

Comment:
@LoliGothick This should be thiserror 101, not anyhow 101, shouldn't it?

Reply:
Fixed on [16ca3ce](https://github.com/LoliGothick/mitama-cpp-result/pull/87/commits/16ca3ce146694956ed7146b25d78c1f2444d0cd0).